.github/workflows/docs.yaml

```diff
@@ -5,14 +5,18 @@ on:
   push:
     branches: [main, master]
 
-name: pkgdown
+name: docs
```

Comment:
Renaming this workflow since we're now building and deploying both Python and R docs at the same time.